NXP MR-VMU-RT1176 flightcontroller update (#3398)
PX4/PX4-user_guide: en/flight_controller/nxp_mr_vmu_rt1176.md

# NXP MR-VMU-RT1176 Flight Controller

:::warning
PX4 does not manufacture this (or any) autopilot.
Contact the [manufacturer](https://www.nxp.com) for hardware support (https://community.nxp.com/) or compliance issues.
:::

:::tip

> This autopilot reference design is [supported](../flight_controller/autopilot_manufacturer_supported.md) by NXP.

Reviewer: Our understanding is this was a PX4 supported reference design. We did al the initial porting etc. It is the standard board design for V6X-RT standard. WE can talk 1:1 if this is not the case.

> This autopilot reference design is [supported](../flight_controller/autopilot_manufacturer_supported.md) by NXP.

Author: @igalloway No, Ramon was asking why you labeled as manufacturer supported, since it is supported. I thought I'd cleaned this up. Will do so now.

  - Flight Controller Module: 38.8 x 31.8 x 14.6mm
  - Standard Baseboard: 50 x 96 x 16.7mm
- Weight
  - Flight Controller Module: 23g
  - Standard Baseboard: 51g

### Interfaces

- 12 PWM servo outputs, 8 with D-SHOT
- R/C input for Spektrum / DSM
- Dedicated R/C input for PPM and S.Bus input
- Dedicated analog / PWM RSSI input and S.Bus output
- 4 general purpose serial ports:
  - 3 with full flow control
  - 1 with separate 1.5A current limit (Telem1)
  - 1 with I2C and additional GPIO line for external NFC reader
- 2 GPS ports
  - 1 full GPS plus Safety Switch Port
  - 1 basic GPS port
- 1 I2C port
- 1 Ethernet port
  - Transformerless Applications
  - 100Mbps
- 1 SPI bus
  - 2 chip select lines
  - 2 data-ready lines
  - 1 SPI SYNC line
  - 1 SPI reset line
- 3 CAN Buses for CAN peripheral (2 currently supported by PX4)
  - CAN Bus has individual silent controls or ESC RX-MUX control
- 2 Power input ports with SMBus
- 1 AD & IO port
- 2 additional analog input
- 1 PWM/Capture input
- 2 Dedicated debug and GPIO lines

- Other Characteristics:
  - Operating & storage temperature: -40 ~ 85°c

## Where to Buy

Order from [NXP](https://www.nxp.com).

## Assembly/Setup

> Wiring is similar to the [Holybro Pixhawk 6X](../flight_controller/pixhawk6x.md#connections) and other boards that follow the [Pixhawk Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).

Reviewer 2: although it is similar to holybros, the key selling point for NXP is the addition of ports, so I suggest NXP invest in creating a wiring diagram

Author: @dk7xe This makes sense to me, though how important this is does depend on how likely this reference design is likely to match the derivatives.

@mrpollo makes a very good point though - I wasn't even aware that the new ports were "significant" from the current documentation. To me this was mostly about the reference design benefits. So if the ports are a key difference, that would be good to capture? 
I suggest that if you agree you do this as a github suggestion.

Reviewer: We won't be able to make a wiring diagram like Holybro does. 
We can create a simpler block diagram

> Wiring is similar to the [Holybro Pixhawk 6X](../flight_controller/pixhawk6x.html#connections) and other boards that follow the [Pixhawk Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).

Author: ```suggestion
Wiring is similar to the [Holybro Pixhawk 6X](../flight_controller/pixhawk6x.md#connections) and other boards that follow the [Pixhawk Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
```

# NXP MR-VMU-RT1176 Flight Controller

<Badge type="tip" text="PX4 v1.15" />

:::warning
PX4 does not manufacture this (or any) autopilot.
Contact the [manufacturer](https://www.nxp.com) for hardware support (https://community.nxp.com/) or compliance issues.
:::

> The _MR-VMU-RT1176_ reference design is based on the Pixhawk<sup>&reg;</sup> FMUv6X-RT open standard, the latest update to the successful family of Pixhawk<sup>&reg;</sup> flight controllers.
> 
> The board includes the same FMU module found on Pixhawk 6X-RT paired with an NXP-based carrier board.
> The carrier board offers 100Base-T1 (two-wire) automotive ethernet, an NFC antenna (Connected to SE051), and a third CAN bus.
> It also removes the IO processor to enable 12 PWM ports, with 8 providing Dshot capability.
> 
> ![MR-VMU-RT1176 Upright Image](../../assets/flight_controller/nxp_mr-vmu-rt1176/mr-vmu-rt1176_upleft.jpg)
> 
> This is NXP's open source _reference design_ for using FMUv6X-RT, and was designed and made in collaboration with industry partners<sup>&reg;</sup>, NXP's mobile robotics team and the PX4 team.
> As a reference/evaluation design, it is intended to be replicated, modified or integrated in production volume by others.
> Several 3rd party manufacturers (such as Holybro.com) provide this or derivative commercial products.
> Note that NXP is a semiconductor vendor, and since it is a reference design, this particular board is not produced in high volume.
> 
> This board takes advantage of multiple Pixhawk​​® open standards, such as the FMUv6X-RT Standard, [Autopilot Bus Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-010%20Pixhawk%20Autopilot%20Bus%20Standard.pdf), and [Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
> Equipped with a high performance NXP i.mx RT1176 dual core Processor, modular design, triple redundancy, temperature-controlled IMU board, isolated sensor domains, delivering incredible performance, reliability, and flexibility.
> 

Author: ```suggestion

The _MR-VMU-RT1176_ reference design is based on the Pixhawk<sup>&reg;</sup> FMUv6X-RT open standard, the latest update to the successful family of Pixhawk<sup>&reg;</sup> flight controllers.

This is NXP's open source _reference design_ for using FMUv6X-RT, and was designed and made in collaboration with industry partners<sup>&reg;</sup>, NXP's mobile robotics team and the PX4 team.
As a reference/evaluation design, it is intended to be replicated, modified or integrated in production volume by others.
It has already gone through all FCC/CE ROHS REACH UKCA, EMI/RFI ESD certifications and is available globally
Several 3rd party manufacturers (such as Holybro.com) provide this or derivative commercial products.

![MR-VMU-RT1176 Upright Image](../../assets/flight_controller/nxp_mr-vmu-rt1176/mr-vmu-rt1176_upleft.jpg)

The board includes the same FMU module found on Pixhawk 6X-RT paired with an NXP-based carrier board.
The carrier board offers 100Base-T1 (two-wire) automotive ethernet, an NFC antenna (Connected to SE051), and a third CAN bus.
It also removes the IO processor to enable 12 PWM ports, with 8 providing Dshot capability.

This board takes advantage of multiple Pixhawk​​® open standards, such as the FMUv6X-RT Standard, [Autopilot Bus Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-010%20Pixhawk%20Autopilot%20Bus%20Standard.pdf), and [Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
Equipped with a high performance NXP i.mx RT1176 dual core Processor, modular design, triple redundancy, temperature-controlled IMU board, isolated sensor domains, delivering incredible performance, reliability, and flexibility.

```

The FMUv6X-RT open standard includes high-performance, low-noise IMUs on board, designed for better stabilization.
Triple-redundant IMU & double-redundant barometer on separate buses.
When PX4 detects a sensor failure, the system seamlessly switches to another to maintain flight control reliability.

An independent LDO powers every sensor set with independent power control.
A vibration isolation System to filter out high-frequency vibration and reduce noise to ensure accurate readings, allowing vehicles to reach better overall flight performances.

External sensor bus (SPI5) has two chip select lines and data-ready signals for additional sensors and payload with SPI-interface, and with an integrated Microchip Ethernet PHY, high-speed communication with mission computers via ethernet is now possible.

The MR-VMU-RT1176 reference design is perfect for developers at corporate research labs, startups, academics (research, professors, students), and commercial application that like to try out T1 (2 wire) automotive ethernet.

> 

Author: Moved this down here
```suggestion

Note that since it is a reference design, this _particular_ board is not produced in high volume.
Similar variants will be available from our licensees.

```

# NXP MR-VMU-RT1176 Flight Controller

> <Badge type="tip" text="PX4 v1.15" />

Author: FYI this badge shows first version it is present in.

:::warning
PX4 does not manufacture this (or any) autopilot.
Contact the [manufacturer](https://www.nxp.com) for hardware support (https://community.nxp.com/) or compliance issues.
:::

> 
> :::tip
> This autopilot reference design is [supported](../flight_controller/autopilot_manufacturer_supported.md) by NXP.
> :::
> 

Author: ```suggestion

```

This is NXP's open source _reference design_ for using FMUv6X-RT, and was designed and made in collaboration with industry partners<sup>&reg;</sup>, NXP's mobile robotics team and the PX4 team.
As a reference/evaluation design, it is intended to be replicated, modified or integrated in production volume by others.
Several 3rd party manufacturers (such as Holybro.com) provide this or derivative commercial products.
Note that NXP is a semiconductor vendor, and since it is a reference design, this particular board is not produced in high volume.

This board takes advantage of multiple Pixhawk​​® open standards, such as the FMUv6X-RT Standard, [Autopilot Bus Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-010%20Pixhawk%20Autopilot%20Bus%20Standard.pdf), and [Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
Equipped with a high performance NXP i.mx RT1176 dual core Processor, modular design, triple redundancy, temperature-controlled IMU board, isolated sensor domains, delivering incredible performance, reliability, and flexibility.

> 

Author: :::tip
This autopilot is [supported](../flight_controller/autopilot_pixhawk_standard.md) by the PX4 maintenance and test teams.
:::
